packages/core-utils/package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "build": "vite build --emptyOutDir",
     "dev": "vite build --mode=development --watch",
-    "check-dts": "check-dts",
+    "check-dts": "check-dts src/**/*",
```

Comment:
`check-dts` started failing after the Vite config was changed to TypeScript. I think because it's outside the files listed in `include` in tsconfig.json.